pnpm-workspaces/BUILD.bazel

```diff
@@ -10,3 +11,10 @@ ts_config(
     visibility = ["//visibility:public"],
     deps = ["//:node_modules/@tsconfig/node16-strictest"],
 )
+
+tsconfig_to_swcconfig.t2s(
```

Comment:
Nice 👍 

I shall now copy this example for my use case 😄

Reply:
yeah let's try it at grammarly